Source/SwiftyJSON.swift

```diff
@@ -314,16 +298,12 @@ public enum Index<T: Any>: Comparable
 public typealias JSONIndex = Index<JSON>
 public typealias JSONRawIndex = Index<Any>
 
-
-extension JSON: Collection
-{
+extension JSON: Swift.Collection {
```

Comment:
It's my fault.
I rebase several commits to one. And the format commit mix in it by mistake